ert/data/record/_transformation.py

```diff
@@ -295,3 +296,31 @@ def _sync_eclsum_transform_output(
             data=dict(zip(map(str, eclsum.dates), map(float, eclsum.numpy_vector(key))))
         )
     return record_dict
+
+
+class CSV2RecordTreeTransformation(RecordTreeTransformation):
+    """Transform dataframe (csv) into a NumericalRecordTree."""
+
+    def __init__(self, columns: List[str]):
+        """
+        Args:
+            columns: List (non-empty) of column names to extract from csv file.
+                Wildcards are not supported.
+        """
+        if not columns:
+            raise ValueError("columns must be non-empty")
+        self._columns = columns
+
+    async def transform_output_sequence(
+        self,
+        mime: str,
+        location: Path,
+    ) -> Tuple[Record, ...]:
+        dframe = pd.read_csv(location, sep=" ")
```

Comment:
Ah, it is not CSV files we are parsing, with `sep=" "`. I think we should parse CSV files as in `pd.read_csv()` without any options, and rather apply a hack upfront to deal with (convert to csv) the space-separated-values file from `semeio`.

Reply:
Agree, I've used sep=" " as I was using the file yielded by the `semeio/design2params` job.